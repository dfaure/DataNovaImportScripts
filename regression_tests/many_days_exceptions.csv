10293A;LE BAN SAINT MARTIN RP;2021-01-13;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2020-12-17;07:00-12:00;14:00-18:15;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-01-14;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-01-12;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2020-12-15;07:00-12:00;14:00-18:15;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-01-16;07:00-12:15;14:15-19:00;;;;;;11:00;11:00;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2020-12-16;07:00-12:00;14:00-18:15;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-01-08;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-01-07;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-01-10;FERME;;;;;;;;;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2020-12-28;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2020-12-31;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-01-11;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-01-15;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-01-09;07:00-12:15;14:15-19:00;;;;;;11:00;11:00;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2020-12-21;07:00-12:00;14:00-18:15;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2020-12-23;07:00-12:00;14:00-18:15;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-01-04;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2020-12-30;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2020-12-18;07:00-12:00;14:00-18:15;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2020-12-22;07:00-12:00;14:00-18:15;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2020-12-26;FERME;;;;;;;;;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2020-12-19;07:00-12:00;14:00-18:15;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-01-02;07:00-12:15;14:15-19:00;;;;;;11:00;11:00;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2020-12-20;FERME;;;;;;;;;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2020-12-29;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-01-03;FERME;;;;;;;;;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-01-01;FERME;;;;;;;;;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-01-05;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2020-12-24;07:00-12:00;14:00-18:15;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-01-06;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2020-12-25;FERME;;;;;;;;;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2020-12-27;FERME;;;;;;;;;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2020-12-14;07:00-12:00;14:00-18:15;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-01-19;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-01-20;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-01-21;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-03-02;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-02-12;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-02-11;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-03-01;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-02-09;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-02-10;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-03-03;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-03-04;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-03-10;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-02-28;FERME;;;;;;;;;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-02-24;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-02-18;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-01-26;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-02-04;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-02-05;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-02-16;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-02-13;07:00-12:15;14:15-19:00;;;;;;11:00;11:00;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-02-22;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-02-20;07:00-12:15;14:15-19:00;;;;;;11:00;11:00;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-02-17;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-02-19;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-02-21;FERME;;;;;;;;;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-02-26;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-02-25;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-02-27;07:00-12:15;14:15-19:00;;;;;;11:00;11:00;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-03-05;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-02-23;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-03-06;07:00-12:15;14:15-19:00;;;;;;11:00;11:00;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-03-07;FERME;;;;;;;;;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-01-22;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-02-08;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-02-03;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-01-25;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-02-02;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-02-06;07:00-12:15;14:15-19:00;;;;;;11:00;11:00;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-01-28;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-03-11;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-01-29;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-02-07;FERME;;;;;;;;;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-02-14;FERME;;;;;;;;;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-02-15;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-02-01;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-01-30;07:00-12:15;14:15-19:00;;;;;;11:00;11:00;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-01-31;FERME;;;;;;;;;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-03-08;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-01-18;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-01-27;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-01-23;07:00-12:15;14:15-19:00;;;;;;11:00;11:00;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-03-09;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-01-17;FERME;;;;;;;;;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-01-24;FERME;;;;;;;;;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-03-12;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-03-17;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-03-14;FERME;;;;;;;;;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-03-15;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-03-13;07:00-12:15;14:15-19:00;;;;;;11:00;11:00;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-03-16;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
10293A;LE BAN SAINT MARTIN RP;2021-03-18;06:15-12:15;14:15-19:00;;;;;;14:30;14:30;;;FERME;;;;;;;
