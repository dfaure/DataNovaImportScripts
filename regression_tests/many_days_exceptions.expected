10293A|LE BAN SAINT MARTIN RP|Mo-Fr 06:15-12:15,14:15-19:00; Sa 07:00-12:15,14:15-19:00; PH off; 2020 Dec 14-19,21-24 07:00-12:00,14:00-18:15; 2020 Dec 26 off
